src/app/file-manager/services/bucket.service.ts

```diff
@@ -2,72 +2,161 @@ import { Injectable } from '@angular/core';
 import { GcsService } from './gcs.service';
 import { Item } from '../models/item';
 import { Observable } from 'rxjs/Observable';
-import { UUID } from 'angular2-uuid';
-import { Type } from '@app/file-manager/models/type';
+import { FirecloudApiService } from '@app/file-manager/services/firecloud-api.service';
```

Comment:
This appears to be unused

Reply:
FirecloudApiService is used in several places.

Reply:
Where is it used in this file? The only occurrence of `FirecloudApiService` in this file that I see is on this import line.

Reply:
got it. I miss-understood your comment.